README.md

```diff
@@ -10,93 +10,22 @@
 
 ## Help Wanted tickets can be found [here](https://github.com/mattermost/mattermost-plugin-mscalendar/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22up+for+grabs%22+label%3A%22help+wanted%22+sort%3Aupdated-desc).
 
-## Table of Contents
+## Contents
 
-1. [Overview](#overview)
-2. [Features](#features)
-3. [Configuration](#configuration)
+- [Overview](#overview)
+- [Features](#features)
+- [Admin Guide](#admin-guide)
 
 ## Overview
 
-This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the Plugin Marketplace and follow [these instructions](#configuration) to install and configure the plugin.
+This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the Plugin Marketplace and follow [these instructions](./docs/admin-guide.md#configuration) to install and configure the plugin.
```

Comment:
We actually recently removed this plugin from the marketplace. We can instead say something like this:

```suggestion
This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version from the [releases page](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and follow [these instructions](./docs/admin-guide.md#configuration) to install and configure the plugin.
```

@fmartingr The "latest" release is a pre-release called `v1.0.0-rc1`. Should we keep this release around?